### ISSUE TITLE
Feature request: Sound and symbol for whippy on surdo

### ISSUE THREAD

**Issue by mattymaxwell** [mooseling/BananaDrum#9]
Not so much a full blown feature request but would be great if we could expand the surdos include a sound for being hit with whippies and maybe even hand?

The need: we have some songs where we do use sounds beyond open and muted on surdo and this is hard to illustrate and hear without adding, for example, a 2nd hep

https://bananadrum.net/?a2=4-4.110.1.1-4.16.07sEY.11IOwQ.2TsK6.380808190.4J2g.5170A2y6.7E.81i5Xg.9YQ~Y

**Comment by mooseling** [comment]
Great suggestion! I have added this to a list of future recordings on [the trello](https://trello.com/b/f6731Frf/banana-drum).

The blocker for this is me getting some surdos home, tuning them to match the existing samples, and recording the whippies. I would really love to record a repique mor too, so perhaps a recording session is a priority.

If you'll allow me to get philosophical, there's a larger question of how many sounds to include in the program. There are so many things you can do! You can hit rims, rods, sides. Different kinds of buzzes. Muted notes, half muted notes. Part of the joy of being a musician is that you aren't a computer, you can do all kinds of creative things. We play a break at LSS that's basically 10 times as good because we all run around the stage whenever we play it. Creative tools always runs the risk of limiting what people will come up with. I digress.... and I need to go to the pub. So I'll stop here.

**Comment by mattymaxwell** [comment]
You're right indeed, even as I was writing the request, my mind was swarming with other sound suggestions and it's definitely hard to decide where the line is
